commit d28a36bbadc360a9409bf9832f1d4171d7c33dad
Author: Koen Kooi <koen@dominion.thruhere.net>
Date:   Wed Jan 26 13:20:20 2011 +0100

    force 'plain' mode

    The new DSS mode breaks XV, so force plain mode

	Upstream-Status: Pending

    Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>

diff --git a/src/omapfb-driver.c b/src/omapfb-driver.c
index 018e040..48aa09c 100644
--- a/src/omapfb-driver.c
+++ b/src/omapfb-driver.c
@@ -326,7 +326,7 @@ OMAPFBPreInit(ScrnInfoPtr pScrn, int flags)
 	OMAPFBProbeController(ofb->ctrl_name);
 
 	/* Do we have the DSS kernel API? */
-	if (stat(SYSFS_DSS_DIR, &st) == 0) {
+	if (0) { //stat(SYSFS_DSS_DIR, &st) == 0) {
 		ofb->dss = TRUE;
 	} else {
 		ofb->dss = FALSE;
